crates/loans/src/benchmarking.rs

```diff
@@ -371,6 +371,10 @@ pub mod benchmarks {
 
     #[benchmark]
     pub fn borrow() {
+        Security::<T>::set_active_block_number(1u32.into());
+        Security::<T>::set_status(StatusCode::Running);
+        assert_ok!(Oracle::<T>::_set_exchange_rate(KBTC, UnsignedFixedPoint::one()));
```

Comment:
Did the benchmarks not already work without this change? Can you move this to a helper utility since many benchmarks seems to share that code already?

Reply:
Nope, this was needed for the interlay testnet runtime. Will move to util func

Reply:
How come it is only needed for that runtime?

Reply:
I'm not sure, but I felt it was not worth spending more time on finding out, especially since we're thinking of removing the testnets. You're very welcome to have a look if you're curious :)